src/Datadog.Trace/Logging/DatadogLogging.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using Datadog.Trace.Vendoring.Serilog;
+using Datadog.Trace.Vendoring.Serilog.Events;
+using Datadog.Trace.Vendoring.Serilog.Sinks.File;
+
+namespace Datadog.Trace.Logging
+{
+    internal static class DatadogLogging
+    {
+        private const string NixDefaultDirectory = "/var/log/datadog/";
+        private static readonly string WindowsDefaultDirectory =
+            Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData), @"Datadog .NET Tracer", "logs");
+
+        private static readonly long? MaxLogFileSize = 10 * 1024 * 1024;
+        private static readonly LogEventLevel MinimumLogEventLevel = LogEventLevel.Verbose; // Lowest level
```

Comment:
This should be configurable, but we can leave that for a separate PR.

Reply:
It is configurable, this is just the default if it's NOT configured.

Reply:
Never mind, then. Thanks!